### DISCUSSION TITLE
Favicons

### DISCUSSION THREAD

**Discussion by johnsicili** [webstudio-is/webstudio-community#54] [Wishlist]
Add the ability to upload a favicon to show up in the browser tabs. 

That's my primary desire, however, it might make sense to knock out all the icon HTML snippets while doing favicons. [Here's a generator](https://realfavicongenerator.net/) that will show many, if not all, of the HTML snippets to add once an icon is uploaded. For example, `<link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">`

**Comment by kof** [comment]
also related issue https://github.com/webstudio-is/webstudio-builder/issues/1915

**Comment by kof** [comment]
This was already launched 🚀
